### PR TITLE
Windows via conda and Doc tidy up

### LINKED ISSUE #8: Update and tidy the docs
There is some markdown formatting, and there are conda install instructions when we aren't on conda yet etc.

### LINKED ISSUE #10: Document how to build on Windows
Here is a draft:

1. Set up the build environment if not already set up (and it's important to setup up MSVC rather than MinGW)
  * Download "Build Tools for Visual Studio 2019" (https://visualstudio.microsoft.com/downloads/)
  * Install, making sure to check the option "C++ build tools" in the installer
2. Set up FFTW
  * Download ZIP file of precompiled FFTW Windows DLLs (http://www.fftw.org/install/windows.html) and extract contents to a folder
  * Open the application "Developer Command Prompt for VS2019", change to the above folder, and run the following commands to create the import libraries (`libfftw3*.lib`):
```
lib /def:libfftw3-3.def /machine:x64
lib /def:libfftw3f-3.def /machine:x64
lib /def:libfftw3l-3.def /machine:x64
```
  * Copy the `libfftw3*.dll` files to somewhere in `%PATH%` (or add the FFTW folder to `%PATH%`)
3. Modify `pyflct/setup_package.py` if not already modified so that the compilation options include the FFTW folder for `include_dirs` and `library_dirs` and the library names for `libraries`.

### LINKED ISSUE #11: Consider changing the license to LGPL
`pyflct` is currently licensed as GPL v3, which could have implications for the license of any package that imports it (but people do disagree).  It contains Berkeley C code ([licensed as LGPL v2](http://cgem.ssl.berkeley.edu/cgi-bin/cgem/FLCT/artifact/7e9001629d1bff57)) and links to FFTW ([nominally licensed as GPL v2](http://www.fftw.org/fftw3_doc/License-and-Copyright.html)).  If FFTW is GPL, it is normally understood that `pyflct` must be GPL because it builds an extension that links to FFTW.

However, FFTW apparently offers a [non-GPL license for purchase](http://www.fftw.org/faq/section1.html#nonfree) as well.  This is the loophole that `pyfftw` uses to [assert a 3-clause BSD license](https://pyfftw.readthedocs.io/en/latest/source/license.html): if the user happens to have a non-GPL license for FFTW, then there is no GPL implication to that user's `pyfftw`.

`pyflct` may want to consider arguing that same – possibly sketchy – loophole to switch to LGPL.  ~(The Berkeley C code will prevent switching to anything more permissive.)~

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #8
Fixes #10
Fixes #11

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -138,29 +142,33 @@ def fix_circleci(x): return None
...
-        'backreferences_dir': str(path.joinpath('generated', 'modules')),
-        'filename_pattern': '^((?!skip_).)*$',
-        'examples_dirs': example_dir,
-        'within_subsection_order': ExampleTitleSortKey,
-        'gallery_dirs': path.joinpath('generated', 'gallery'),
-        'default_thumb_file': path.joinpath('logo', 'sunpy_icon_128x128.png'),
-        'abort_on_example_error': False,
-        'plot_gallery': True,
-        'doc_module': ('sunpy')
+        "backreferences_dir": str(path.joinpath("generated", "modules")),
```
Comment (Cadair) [inline docs/conf.py:150]: you are just causing chaos when we retemplate this now...

File: setup.cfg [hunk 2]
```diff
@@ -2,18 +2,19 @@
...
+license = LGPL-2.1
 license_file = LICENSE.rst
 url = https://docs.sunpy.org/projects/pyflct/
 description = A Python wrapper for Fourier Local Correlation Tracking.
 long_description = file: README.rst
 classifiers =
   Development Status :: 5 - Production/Stable
   Intended Audience :: Science/Research
-  License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+  LICENSE :: OSI APPROVED :: GNU LESSER GENERAL PUBLIC LICENSE V2.1 (LGPLV2.1)
```
Comment (Cadair) [inline setup.cfg:13]: why so much caps

File: setup.cfg [hunk 3]
```diff
@@ -50,9 +52,6 @@ pyflct = data/*
 [tool:pytest]
 testpaths = "pyflct" "docs"
 norecursedirs = ".tox" ".eggs" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" ".history"
-doctest_plus = enabled
```
Comment (Cadair) [inline setup.cfg:53]: why no doctest?

File: tox.ini [hunk 4]
```diff
@@ -3,36 +3,13 @@ envlist =
...
-    tox-pypi-filter >= 0.10
 
 [testenv]
-# The following option combined with the use of the tox-pypi-filter above allows
-# project-wide pinning of dependencies, e.g. if new versions of pytest do not
-# work correctly with pytest-astropy plugins. Most of the time the pinnings file
-# should be empty.
-pypi_filter_requirements = https://raw.githubusercontent.com/sunpy/package-template/master/.test_package_pins.txt
-# Pass through the following environemnt variables which may be needed for the CI
-passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI TRAVIS
```
Comment (Cadair) [inline tox.ini:20]: we need to add CIRCLECI to this on the template.

Again with the chaos causing when we retemplate though.

File: docs/conf.py (hunk 1 above)
Comment (nabobalis) [reply docs/conf.py:150]: Black your code.

File: setup.cfg (hunk 3 above)
Comment (nabobalis) [reply setup.cfg:53]: No doctests.

File: tox.ini (hunk 4 above)
Comment (nabobalis) [reply tox.ini:20]: I can revert these changes.

File: setup.cfg (hunk 2 above)
Comment (nabobalis) [reply setup.cfg:13]: I will fix it.

File: tox.ini (hunk 4 above)
Comment (nabobalis) [reply tox.ini]: Fixed.

File: setup.cfg (hunk 2 above)
Comment (nabobalis) [reply setup.cfg]: Fixed.

**Pull request merged**